blocks/hero-banner/hero-banner.css

```diff
@@ -69,6 +69,10 @@
     position: relative;
 }
 
+.hero-banner.image-fit img {
+    height: 15rem;
+}
+
```

Comment:
This seems to be very specific style or do you think it could be useful at other places.

Reply:
Personally, I think we have/will have the similar case that using desktop image for mobile view,